packages/dev/core/src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -346,6 +396,21 @@ export class RenderTargetTexture extends Texture implements IRenderTargetTexture
         return this._renderTarget?._depthStencilTexture ?? null;
     }
 
+    /**
+     * Instantiate a render target texture. This is mainly used to render of screen the scene to for instance apply post process
+     * or used a shadow, depth texture...
+     * @param name The friendly name of the texture
+     * @param size The size of the RTT (number if square, or {width: number, height:number} or {ratio:} to define a ratio from the main scene)
+     * @param scene The scene the RTT belongs to. The latest created scene will be used if not precised.
+     * @param options The options for creating the render target texture.
+     */
+     constructor(
+        name: string,
+        size: number | { width: number; height: number; layers?: number } | { ratio: number },
+        scene?: Nullable<Scene>,
+        options?: Partial<RenderTargetTextureOptions>

```

Comment:
Wouldn't it be better to flag all properties of `RenderTargetTextureOptions` as optional (with `?:`) instead of using `Partial` here?

If someone wants to do:
```typescript
const myOptions: RenderTargetTextureOptions = {
    ...
};

// create the RTT using these options
```
you will need to set a value for all the properties.

Reply:
I was playing around and should have reverted.